UM/Logger.py

```diff
@@ -28,8 +30,15 @@ def getLoggers(cls):
     #   \param *args \type{list} List of variables to be added to the message.
     @classmethod
     def log(cls, log_type, message, *args):
+        function = inspect.currentframe().f_back.f_code
+        filename = function.co_filename
+        for path in sys.path:
+            if filename.startswith(path):
+                filename = filename.replace(path, "<PATH>")
```

Comment:
So, how do you want it replaced now? Should I better take a ```#```? It would make it less confusing then ```~```. ```:``` looks somehow ugly :sweat_smile: 
What do you think, @nallath @Ghostkeeper ?